src/js/utils/browser.js

```diff
@@ -192,6 +192,15 @@ export const IS_SAFARI = (/Safari/i).test(USER_AGENT) && !IS_CHROME && !IS_ANDRO
  */
 export const IS_ANY_SAFARI = (IS_SAFARI || IS_IOS) && !IS_CHROME;
 
+/**
+ * Whether or not this is a Windows machine.
+ *
+ * @static
+ * @const
+ * @type {Boolean}
+ */
+export const IS_WINDOWS = (/Windows/i).test(USER_AGENT);
```

Comment:
unfortunately, this makes this a minor update

Reply:
Darn!